src/aat/resources/features/F-200 - New Access Control Create Case/S-200.1.td.json

```diff
@@ -0,0 +1,57 @@
+{
+  "_guid_": "S-200.1",
+
+  "productName": "CCD Data Store",
```

Comment:
Lets also create a F-200_CreateCase_Base file with the common lines of code and then have all the data files for each scenario extend that file